test/integration/simple-deployment/00-assert.yaml

```diff
@@ -1,3 +1,12 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 30
+collectors:
+  - type: pod
+    selector: app=test
+  - type: command
+    command: kubectl logs -l app=test -n $NAMESPACE
```

Comment:
the same goes here, also here we need to check that the crds have been created so a log of the crd of the namespace could be enough